gravity-forms/gw-update-posts.php

```diff
@@ -65,6 +66,10 @@ public function update_post_by_entry( $entry, $form ) {
 			$post->post_author = (int) rgar( $entry, $this->_args['author'] );
 		}
 
+		if ( $this->_args['status'] ) {
+			$post->post_status = rgar( $entry, $this->_args['status'] );
+		}
```

Comment:
It'd be awesome if we could get a list of [registered post statuses](https://developer.wordpress.org/reference/functions/get_post_stati/) and then make sure that the user-selected (or enetered) status is a valid post status before we attempt to set it. If not, we could just skip updating it.